WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -330,6 +333,22 @@ public void onClick(View v) {
         mInfoButtonSecondary.setOnClickListener(infoButtonListener);
     }
 
+    /*
+     * autofill the username and password from BuildConfig/gradle.properties (developer feature)
+     */
+    private void autofillFromBuildConfig() {
+        String userName = BuildConfig.DOTCOM_LOGIN_USERNAME;
+        String password = BuildConfig.DOTCOM_LOGIN_PASSWORD;
```

Comment:
We should do some extra work here to get these by reflection so we can make `wp.dotcom_login_username` and `wp.dotcom_login_password` optional in the property file.